activerecord/test/cases/arel/table_test.rb

```diff
@@ -190,21 +190,28 @@ class TableTest < Arel::Spec
     describe "equality" do
       it "is equal with equal ivars" do
         relation1 = Table.new(:users)
```

Comment:
Let's change this to `Table.new(:users, as: "zomg")` and the same below

Reply:
Done.